pg_shard--1.1.sql

```diff
@@ -41,6 +41,12 @@ CREATE SCHEMA pgs_distribution_metadata
 	CREATE SEQUENCE shard_id_sequence MINVALUE 10000 NO CYCLE
```

Comment:
Yeah; I'm in the process of rebasing just the schema changes from #103 on top of this branch so it will be much more clear what the behavior will be.

There obviously _could_ be collisions when upgrading from PostgreSQL + `pg_shard` to CitusDB, but that's a pretty obscure edge case, IMO.